### ISSUE TITLE
[baseline|hash-library|czmq] File conflict

### ISSUE THREAD

**Issue by Neumann-A** [microsoft/vcpkg#24455]
```
Uploaded binaries to 1 HTTP remotes.
Installing package hash-library[core]:arm64-windows...
The following files are already installed in D:/installed/arm64-windows and are in conflict with hash-library:arm64-windows

Installed by czmq:arm64-windows
    include/sha1.h

Elapsed time for package hash-library:arm64-windows: 2.75 s
```

**Comment by Thomas1664** [comment]
Fixed in #24433

**Comment by BillyONeal** [comment]
Fixed in https://github.com/microsoft/vcpkg/pull/24434
